scripts/common/prompts.sh

```diff
@@ -88,16 +65,11 @@ function prompt_for_token() {
     if [ -n "$KUBEADM_TOKEN" ]; then
         return
     fi
-    if ! prompts_can_prompt ; then
-        bail "kubernetes.kubeadmToken required"
```

Comment:
i do not understand why this is changing

Reply:
Because instead of `read PROMPT_RESULT < /dev/tty` we are using the read function `read -p "Kubernetes join token:" TOKEN_PROMPT_RESULT` (see the line 70).

Note that `prompts_can_prompt ` will fail always with tee.